tools/cli/domain_commands.go

```diff
@@ -574,7 +574,6 @@ EmitMetrics: {{.EmitMetrics}}
 IsGlobal(XDC)Domain: {{.IsGlobal}}
 ActiveClusterName: {{.ActiveCluster}}
 IsActiveActiveDomain: {{.IsActiveActiveDomain}}
-ActiveClustersByRegion: {{.ActiveClustersByRegion}}
```

Comment:
I don't really think it's worthwhile putting the (likely quite large) cluster attributes here.